src/Database/PostgreSQL/PQTypes/SQL/Builder.hs

```diff
@@ -443,6 +452,16 @@ checkAndRememberMaterializationSupport = do
     fetchOne runIdentity
   liftIO $ writeIORef withMaterializedSupported (isRight res)
 
+-- This function has to be called as one of first things in your program
```

Comment:
Isn't this feature supported in pretty much all PostgreSQL versions? I checked and 9.6 looks to have it.

Reply:
They started in 8.4 according to Wikipedia, but I don't know what are the older versions supported by hpqtypes-extra. If we don't need to check for the feature, I'll be glad to remove these !

Reply:
Removed then.